src/components/views/ApplicationViews.js

```diff
@@ -34,26 +36,45 @@ export const ApplicationViews = () => {
         path="/"
         element={
           <AuthorizedRoute url={location.pathname}>
-            <NavBar loggedInUser={loggedInUser} setLoggedInUser={setLoggedInUser} />
+            <NavBar
+              loggedInUser={loggedInUser}
+              setLoggedInUser={setLoggedInUser}
+            />
             <Outlet />
           </AuthorizedRoute>
-        }>
-        <Route index element={<>Rare - Home Page</>} /> {/* home page will go here */}
-        <Route path="/categories">
-          <Route index element={<CategoryList />} />
-          <Route path="create" element={<CategoryForm />} />
-        </Route>
-        <Route path="/tags">
-          <Route index element={<TagList />} />
-          <Route path="create" element={<CreateTagForm />} />
-        </Route>
+        }
+      >
+        <Route index element={<>Rare - Home Page</>} />{" "}
+        {/* home page will go here */}
+        <Route path="/category-list" element={<CategoryList />} />
+        <Route path="/createCategory" element={<CategoryForm />} />
+        <Route path="/tags" element={<TagList />} />
+        <Route path="/tags/create" element={<CreateTagForm />} />
```

Comment:
I noticed that I cannot currently access the **Category Manager** view, or the **Category Creation** view. Upon further inspection, it looks like you may have accepted the wrong result when dealing with merge conflicts in `ApplicationViews.js`. The current implementation of `main` includes nested routes for `/categories` and `/tags` in order to support the `/tags/create` and `/categories/create` views. But in your branch, this is what I see when looking at the file changes:

<img width="931" alt="Screen Shot" src="https://github.com/NSS-Day-Cohort-68/rare-client-rare-team-4/assets/147445675/6428ca41-a145-41dd-8caa-b41b601ddfc2">

Lines 42-49 (red) should replace lines 49-52 (green) in order to fix this, and revert this change.